dist/about.html

```diff
@@ -28,69 +28,72 @@
       </nav>
-      <a href="#" class="menu-nav__btn"> CC Campaign </a>
+      <a href="https://anilist.co/user/Zeraltz/" class="menu-nav__btn"> AniList </a>
     </div>
   </header>
   <main>
     <section class="about-hero lato_regular">
-      <p class="about-hero__quote">Hello! Sharing World!</p>
-      <h1 class="about-hero__title cocogoose_regular">CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
+      <p class="about-hero__quote">"Hello! Anime Fans!"</p>
+      <h1 class="about-hero__title cocogoose_regular">ANIMATION FANS ALL OVER THE WORLD! GATHER!</h1>
       <hr class="desktop about-divider">
-      <p class="about-hero__text-content">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Cum nam distinctio, quaerat aspernatur quos iusto ipsa, voluptatem sunt aut, nulla sit accusantium magni itaque nesciunt unde dolorem minus possimus at deleniti optio dicta!</p>
-      <p class="about-hero__contact-us">Please contact us per Email for any further questions about CC Global Summit 2015 <span class="about-hero__contact-us__email">ccsummit2015@cckorea.org</span></p>
+      <p class="about-hero__text-content">I am a 27 seven years old half Hispanic, half Asian guy from South America, I have been watching anime consistently since 2009. In all these years I have watched more than 400 shows (adding up to 3100 hours) and I usually critique them based on facts, general consensus and my own feelings. I like writing code in my free time, and also writing novels (I'm bad at this one tho).
+      I speak English and Spanish, I tried to learn German by myself but I gave up, and now learning Chinese by myself as well.
+    
+      Right now I'm studying to become a Full Stack Web Developer, and this is one of my projects, THANK YOU FOR BEING PART OF IT</p>
+      <p class="about-hero__contact-us">Please contact me per Email for any further questions about Animation Fans 2022 <span class="about-hero__contact-us__email">info@animationfans.com</span></p>
       
     </section>
     <hr class="anifans-logo__divider">
     <section class="anifans-logo lato_regular">
-      <h2 class="anifans-logo__title">Creative Commons Global Summit 2015 Logo</h2>
+      <h2 class="anifans-logo__title">Animation Fans Logo Explained</h2>
       <hr class="about-divider">
       <p class="anifans-logo__text-content">
-        Lorem ipsum dolor sit amet consectetur adipisicing elit. Hic vero molestiae, vitae veritatis ipsa obcaecati est corporis architecto temporibus nisi.
+        For the logo I just went for two single concepts, minimalism and cuteness. Everyone, or almost everyone, love cute things, so I decided to go for something that was recognizable and cute, thus, I picked a "moe" anime girl to be the face of the site for the time being, also, a modern website/brand needs a modern logo, so I went for a minimalist, simple and easy design, using Montserrat font for it.
       </p>
       <img src="./img/logo/logo-black.png" alt="" class="anifans-logo__logo">
```

Comment:
- [ ] Kindly add descriptive text to the `alt` attribute as this makes your page more accessible in case of image load failure.

Reply:
Fixed